desktop/main/startApp.ts

```diff
@@ -166,16 +171,16 @@ const startApp = (): AppStore => {
     handleAppWalletChange($isWalletActivated, $wallet, $managers),
     // Update currentLayer & rootHash
     // Update networks on init
-    fetchDiscovery($networks),
+    fetchDiscovery($networks, $mainWindow),
     // Update networks each N seconds
-    fetchDiscoveryEach(60 * MINUTE, $networks),
+    fetchDiscoveryEach(60 * MINUTE, $networks, $mainWindow),
     // And update them by users request
-    listNetworksByRequest(),
+    listNetworksByRequest($mainWindow),
```

Comment:
This is a good clue that something should be refactored. So I wrote what and how I think it should be done in other comments. However, I want to share my thoughts on it. Might be helpful further :)

Some actions that should not end in any fs interaction now depend on `$mainWindow` without any reason...

So, we can say that there might be other errors (like fetching failed with some error and we want to show some notification to the User besides the empty list), but in this case, seems we need another approach there.

In this case, I'd collect such notifications in some `Rx.Subject<UserNotification>` (so it will be passed as arguments there), and then handle new notifications (and send them to MainWindow) in one single place, e.g.
```
fetchDiscovery($networks, $notifications),
fetchDiscoveryEach(60 * MINUTE, $networks, $notifications),
sendNotifications($mainWindow, $notifications),
```.

However, this is not our current case.
So let's stick to the proposed solution in another comment ;)